src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -365,7 +396,11 @@ public void faceBlock(@Nullable BlockPos block)
         double squareDifference = Math.sqrt(xDifference * xDifference + zDifference * zDifference);
         double intendedRotationYaw = (Math.atan2(zDifference, xDifference) * 180.0D / Math.PI) - 90.0;
         double intendedRotationPitch = -(Math.atan2(yDifference, squareDifference) * 180.0D / Math.PI);
-        this.setRotation((float) this.updateRotation(this.rotationYaw, intendedRotationYaw, 30), (float) this.updateRotation(this.rotationPitch, intendedRotationPitch, 30));
+        this.setRotation((float) updateRotation(this.rotationYaw, intendedRotationYaw, 30), (float)updateRotation(this.rotationPitch, intendedRotationPitch, 30));
+
+        double goToX = xDifference > 0? MOVE_MINIMAL : -MOVE_MINIMAL;
+        double goToZ = zDifference > 0? MOVE_MINIMAL : -MOVE_MINIMAL;
+        moveEntity(goToX, 0, goToZ);
```

Comment:
Okay, because Minecraft doesn't rerender the position when he doesn't move, in order to have a worker face a certain position we have to make him move - even when it's only 0.01